manim/utils/opengl.py

```diff
@@ -31,11 +31,11 @@ def orthographic_projection_matrix(
         return projection_matrix
 
 
-def perspective_projection_matrix(width=None, height=None, near=4, far=18, format=True):
+def perspective_projection_matrix(width=None, height=None, near=2, far=30, format=True):
```

Comment:
What does these `near=2` and `far=30` mean? Can you add a comment on how did you come up with these defaults?

Reply:
I just tried numbers until I found something that fit the 2D dimensions pretty closely.